### PR TITLE
[RDY] feat(infra): Add infrastructure code with terraform

### PR DESCRIPTION
**Pull request by deuzu**

### CODE REVIEW

**Comment by hanfi** [comment]
please no cloudformation :'( it sucks 
terraform would be a better option

**Comment by deuzu** [comment]
@hanfi Please explain.

File: github-democrat/main.tf [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Provider
```
Comment (deuzu) [inline github-democrat/main.tf]: should move to another repository

**Comment by hanfi** [comment]
Amazing you are the best

**Pull request merged**